services/wallet/router/router.go

```diff
@@ -111,6 +121,12 @@ func (r *Router) GetPathProcessors() map[string]pathprocessor.PathProcessor {
 	return r.pathProcessors
 }
 
+func (r *Router) SetTestBalanceMap(balanceMap map[string]*big.Int) {
```

Comment:
we really need to stop doing this sort of thing in favor of mocked dependencies...

Reply:
That's not done here, that is the same thing that was done 2-3 months ago, I just had to create that function cause I moved the code around.

Reply:
And I agree, potentially we should get rid of that, but somehow we still haven't come to it.